src/webextension/background/message-ports.js

```diff
@@ -143,7 +143,10 @@ export default function initializeMessagePorts() {
     case "signout":
       return openDataStore().then(async (datastore) => {
         // TODO: perform (light) signout from FxA
-        const account = await getAccount();
+        const account = getAccount();
+
+        await closeView("manage");
```

Comment:
nit: generally we ought to hold off on closing until the action is done (see other handlers herein), but it's a nit.